packages/kbn-guided-onboarding/src/components/landing_page/guide_card_footer.tsx

```diff
@@ -48,17 +48,23 @@ const progressBarLabelCss = css`
 
 export interface GuideCardFooterProps {
   guides: GuideState[];
-  useCase: UseCase;
-  activateGuide: (useCase: UseCase, guideState?: GuideState) => void;
+  useCase: GuideCardUseCase;
+  telemetryId: string;
+  activateGuide: (useCase: GuideCardUseCase, guideState?: GuideState) => void;
 }
-export const GuideCardFooter = ({ guides, useCase, activateGuide }: GuideCardFooterProps) => {
+export const GuideCardFooter = ({
+  guides,
+  useCase,
+  telemetryId,
+  activateGuide,
+}: GuideCardFooterProps) => {
   const guideState = guides.find((guide) => guide.guideId === (useCase as GuideId));
   const viewGuideButton = (
     <EuiFlexGroup justifyContent="center">
       <EuiFlexItem grow={false}>
         <EuiButton
           // Used for FS tracking
-          data-test-subj={`onboarding--guideCard--view--${useCase}`}
+          data-test-subj={`onboarding--guideCard--view--${telemetryId}`}
```

Comment:
This should also be OK. We are considering "start" when the first step is clicked from the guide panel; we are not tracking the card click for 8.6.